selfdrive/car/gm/gmcan.py

```diff
@@ -58,15 +58,13 @@ def create_gas_regen_command(packer, bus, throttle, idx, acc_engaged, at_full_st
 
   return packer.make_can_msg("ASCMGasRegenCmd", bus, values)
 
-def create_friction_brake_command(packer, bus, apply_brake, idx, near_stop, at_full_stop):
+def create_friction_brake_command(packer, bus, apply_brake, idx, near_stop, at_full_stop, intense_braking):
+  mode = 0x1 # Default brake mode
+  if intense_braking:
+    mode = 0xa # intense braking requested (>~80ish)
 
-  if apply_brake == 0:
-    mode = 0x1
-  else:
-    mode = 0xa
-
-    if at_full_stop:
-      mode = 0xd
+  if want_full_stop:
```

Comment:
Undefined 'want_full_stop'
Looks like this and "want full stop" PRs should be a single PR, to avoid merge conflicts like that.